man/sd_notify.xml

```diff
@@ -258,16 +258,24 @@
         <term>BUSERROR=…</term>
 
         <listitem><para>If a service fails, the D-Bus error-style error code. Example:
-        <literal>BUSERROR=org.freedesktop.DBus.Error.TimedOut</literal></para>
+        <literal>BUSERROR=org.freedesktop.DBus.Error.TimedOut</literal>. Note that this assignment is
+        currently not used by <command>systemd</command>.</para>
 
         <xi:include href="version-info.xml" xpointer="v233"/></listitem>
       </varlistentry>
 
       <varlistentry>
         <term>EXIT_STATUS=…</term>
 
-        <listitem><para>If a service exits, the return value of its <function>main()</function> function.
-        </para>
+        <listitem><para>The exit status of a service or the manager itself. Note that
+        <command>systemd</command> currently does not consume this value when sent by services, so this
```

Comment:
isn't this also propagated in a dbus signal? ie, external clients could also consume it

Reply:
I don't see this happening:
```console
$ git grep -n '\bEXIT_STATUS[^_]' src/
# those three send the status
src/core/main.c:3201:        (void) sd_notifyf(0, "EXIT_STATUS=%i", retval);
src/shared/main-func.h:24:                (void) sd_notifyf(0, "EXIT_STATUS=%i", ret);            \
src/shutdown/shutdown.c:587:        (void) sd_notifyf(0, "EXIT_STATUS=%i", arg_exit_code);

# this is in sanitize_environment() call to strv_env_unset_many()
src/core/manager.c:639:                        "EXIT_STATUS",

# this is the call to log_struct()
src/core/unit.c:5927:                        "EXIT_STATUS=%i", status,

# false postive
src/shared/exit-status.h:81:        EXIT_STATUS_LIBC    = 1 << 0,  /* libc EXIT_STATUS/EXIT_FAILURE */
```
As I wrote, it seems we never consume the value.